examples/config_github.js

```diff
@@ -0,0 +1,181 @@
+// This config file is used during the github pipeline
```

Comment:
This could go into the .github directory or into a test directory with the playwirght stuff. 

For this test file, you could removed the commented out stuff.